PorticoExportDom.inc.php

```diff
@@ -95,10 +95,11 @@ private function _buildArticle() : DOMElement {
 		$journalTitleGroupNode->appendChild($doc->createElement('journal-title', $journal->getLocalizedPageHeaderTitle()));
 
 		// issn
-		foreach (['printIssn', 'issn', 'onlineIssn'] as $name) {
+		foreach (['printIssn' => 'print', 'onlineIssn' => 'online-only'] as $name => $format) {
```

Comment:
I'm considering the `issn` setting has been removed since a long time ago.

Reply:
:+1: